client/layout/guided-tours/tours/checklist-publish-post-tour.js

```diff
@@ -0,0 +1,134 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { noop } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Next,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const ChecklistPublishPostTour = makeTour(
+	<Tour name="checklistPublishPost" version="20171205" path="/non-existent-route" when={ noop }>
+		<Step
+			name="init"
+			placement="right"
+			style={ {
+				animationDelay: '0.7s',
+			} }
+		>
+			<p>
+				{ translate(
+					'It’s time to get your blog rolling with your first post. ' +
+						'Let’s welcome your audience with a brief introduction describing what your blog is about ' +
+						'and how often you intend on posting.'
+				) }
+			</p>
+			<ButtonRow>
+				<Next step="categories-tags">{ translate( 'All done, continue' ) }</Next>
+				<SiteLink href="/checklist/:site">{ translate( 'Return to the checklist' ) }</SiteLink>
+			</ButtonRow>
+		</Step>
+
+		<Step
+			name="categories-tags"
+			target="accordion-categories-tags"
+			arrow="right-top"
+			placement="beside"
+		>
+			<p>
+				{ translate(
+					'Categories and Tags not only help organize your content but also bring people to your site via the Reader. ' +
+						'Pick one or two descriptive tags about your post for it to start showing up to other users in the Reader.'
+				) }
+			</p>
+			<ButtonRow>
+				<Next step="featured-images">{ translate( 'All done, continue' ) }</Next>
+				<SiteLink href="/checklist/:site">{ translate( 'Return to the checklist' ) }</SiteLink>
+			</ButtonRow>
+		</Step>
+
+		<Step
+			name="featured-images"
+			target="editor-featured-image-current-image"
+			arrow="top-left"
+			placement="below"
+		>
+			<Continue target="editor-featured-image-current-image" step="choose-image" click>
+				<p>
+					{ translate(
+						'Featured images are a great way to add more personality to your pages. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Featured images are a great way to add more personality to your pages. Let's add something a little more relevant to your About page text.' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/first-post-tour)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).